code42_connector.py

```diff
@@ -28,6 +31,19 @@ def __init__(self):
         self._username = None
         self._password = None
         self._client = None
+        self._action_map = {
```

Comment:
I'm doing something similar to this in my ci/cd branch im working on :-)

Reply:
Could we do this and not have to have any maps of identifiers to methods? 

```
ret_val = getattr(self, "_handle_{0}".format(action_id))(param)
```

Reply:
you could, but that would create a strict dependency on what handler functions now _have_ to be named. It sometimes isn't a big deal, but I would generally discourage this kind of coupling.